README.md

```diff
@@ -1,30 +1,40 @@
 
 ## Installation
+
 (will make available via `pip install` and `conda install` when project is closer to completion - hit the Watch button if you'd like to be updated on progress!)
 
 ## Usage
-(also coming soon - check back in a few weeks!)
+
+`python main.py` to train an English model and save per-epoch checkpoints.
+Add `-lang ru` to train a Russian model.
+
+Once you have a usable model checkpoint, open `python/demo.py` and set the path to the 
+model checkpoint you'd like to use and the demo's default language. Then run `streamlit run demo.py`, which opens a web app you can use to test the model's predictions interactively.
 
 ## Requirements
-- **NOTE**: Since there are quite a few requirements, we recommend creating a virtual environment when using this package.
-- pytorch-transformers (1.1.0+) (and torch)
-- spaCy (2.1.0+)
-- keras (and TensorFlow) (only used for utility functions - will try to eliminate this dependency down the line)
-- numpy, pandas
-- pre-commit
-- (Any packages used for visualizing datasets and/or model performance in the frontend - will update once frontend is ready)
-- (Probably Flask and any other deployment dependencies - will update once ready)
-- Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
+
+-   **NOTE**: Since there are quite a few requirements, we recommend creating a virtual environment when using this package. (use `requirements.txt` when creating environment from scratch)
+-   pytorch-transformers 1.1.0+, torch 1.2.0+
+-   spaCy (2.1.0+)
+-   TensorFlow 1.14.0+, keras 2.2.4+ (only used for utility functions - will try to eliminate this dependency down the line)
+-   numpy 1.15.4+, pandas 0.25.0+
+-   pre-commit 1.18.0+
+-   streamlit 0.4.0+
+-   (Deployment dependencies - TBD)
+-   Miscellaneous minor packages: pathlib, seqeval, tqdm, yaml
 
 ## Models
-- The English model is a fine-tuned implementation of Google's `bert-base-cased` model, ensembled with spaCy's `en_core_web_lg`, which uses a CNN architecture.
-- The Russian model is a fine-tuned implementation of Google's `bert-base-multilingual-cased` model, ensembled with spaCy's multilingual `xx_ent_wiki_sm` NER model, which uses a CNN architecture.
+
+-   The English model is a fine-tuned implementation of Google's `bert-base-cased` model, ensembled with spaCy's `en_core_web_lg`, which uses a CNN architecture.
+-   The Russian model is a fine-tuned implementation of Google's `bert-base-multilingual-cased` model, ensembled with spaCy's multilingual `xx_ent_wiki_sm` NER model, which uses a CNN architecture.
 
 ## Data Sources
+
```

Comment:
missing information about the spacy model's data sources

Reply:
Good point, will add